### PR TITLE
update rails example

### PR DESCRIPTION
**Pull request by MiguelMoll**

### CODE REVIEW

**Comment by MiguelMoll** [comment]
[rails example](https://trello.com/c/Ld4Djtew/2461-rails-example)

File: examples/rails/Gemfile [hunk 1]
```diff
@@ -3,7 +3,8 @@ source 'https://rubygems.org'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '4.2.6'
-# Use sqlite3 as the database for Active Record
+# Use postgres as the database for Active Record
+gem 'pg'
 gem 'sqlite3'
```
Comment (mattmanning) [inline examples/rails/Gemfile]: I would remove the sqlite3 gem since it's not being used here.

Comment (MiguelMoll) [inline examples/rails/Gemfile]: Believe @nzoschke added this back as some of the tools were complaining. Perhaps he can explain why better than me.

Comment (nzoschke) [inline examples/rails/Gemfile]: config/database.yml references sqlite, so tools like `rails g model` complain without the Gem.

Could also update the config

**Pull request merged**